src/Engine/Rendering/Visualizers/OrbitVisualizer.cpp

```
@@ -0,0 +1,105 @@
+		// Bind orbit vertex buffer
+		VkBuffer vertexBufs[] = { m_orbitVertBuffer->buffer };
+		VkDeviceSize vertBufOffsets[] = { 0 };
+		vkCmdBindVertexBuffers(
+			m_secondCmdBufs[frameIdx], 
+			0, 1, vertexBufs, vertBufOffsets
+		);
+
+
+		// Bind global UBO
+		uint32_t globalUBOOffset = static_cast<uint32_t>(m_alignedGlobalUBOSize);
+		vkCmdBindDescriptorSets(
```

Cppcheck (2.20.0)

[performance] Function parameter 'queueFamilies' should be passed by const reference.

(passedByValue)